cmake/globalutils.cmake

```diff
@@ -862,6 +862,26 @@ function(ivw_get_target_property_recursive retval target property alsoInterfaceT
     set(${retval} ${res} PARENT_SCOPE)
 endfunction()
 
+
+#--------------------------------------------------------------------
+# Get the list of targets the given target depends on. 
+function(ivw_get_target_depending_on retval target alsoInterfaceTargets)
```

Comment:
why not just use ivw_get_target_property_recursive with INTERFACE_LINK_LIBRARIES